test/test_buffer.cpp

```diff
@@ -161,32 +168,86 @@ namespace sparrow
             CHECK_EQ(b3.data()[idx], expected_value);
         }
 
-        TEST_CASE("equality comparison")
+        // Iterators
+
```

Comment:
Missing tests: static check that
- for a given set of T, buffer_view<T> is a contiguous range
- buffer_view<T>::iterator is the appropriate type of iterator

Reply:
Actually, I dont see any test specific to buffer_view?

Reply:
These are the tests of the `buffer` class; the test suite of `buffer_view` now starts at line 315 and remains unchanged.